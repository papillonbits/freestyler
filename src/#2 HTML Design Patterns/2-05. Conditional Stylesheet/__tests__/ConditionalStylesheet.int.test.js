import { regular } from './ConditionalStylesheet.int.story'

describe('<ConditionalStylesheet />', () => {
  describe('Render', () => {
    test('must match regular()', () => {
      expect(global.renderToJSON(regular())).toMatchSnapshot()
    })
  })
})
