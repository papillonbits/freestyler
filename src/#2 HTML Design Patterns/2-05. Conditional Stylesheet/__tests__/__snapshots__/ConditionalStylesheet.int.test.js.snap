// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ConditionalStylesheet /> Render must match regular() 1`] = `
<div>
  <div
    class="conditional-stylesheet"
  >
    <h1>
      Conditional Stylesheet
    </h1>
    <p
      class="conditional-stylesheet__test"
    >
      In Internet Explorer 6, this box has a border and background. Look at the README for the full document
    </p>
  </div>
</div>
`;
